tests/integration/targets/hcloud_datacenter_info/tasks/main.yml

```diff
@@ -8,7 +8,7 @@
 - name: verify test gather hcloud datacenter infos
   assert:
     that:
-      - hcloud_datacenters.hcloud_datacenter_info| list | count == 4
+      - hcloud_datacenters.hcloud_datacenter_info| list | count == 5
```

Comment:
does it maybe make sense to make `>=5` here? So we don't need to change it when we might add new datacenters/locations